src/Login/index.jsx

```diff
@@ -0,0 +1,148 @@
+import React, { Component, PropTypes } from 'react';
+import { themr } from 'react-css-themr';
+import { connect } from 'react-redux';
+import {
+    addLocaleData,
+    IntlProvider,
+    FormattedMessage,
+    defineMessages
+} from 'react-intl';
+import enLocaleData from 'react-intl/locale-data/en';
+import zhLocaleData from 'react-intl/locale-data/zh';
+
+import { setUsername, setPassword, toggleRemember, attemptLogin } from './actions';
+import Input from '../Form/Input';
+import style from './style.scss';
+import constants from '../constants';
+import zhMessage from './l10n/zh_CN';
+import enMessage from './l10n/en_US';
+
+addLocaleData(enLocaleData);
+addLocaleData(zhLocaleData);
+
+const getLocaleMessage = (locale) => {
+    switch (locale) {
+        case 'en':
+            return enMessage;
+        case 'zh':
+            return zhMessage;
+    }
+}
+
+const placeholderMessages = defineMessages({
+    username: {
+        id: 'placeholder.username'
+    },
+    password: {
+        id: 'placeholder.password'
+    }
+})
+
+class Login extends Component {
+    static propTypes = {
+        theme: PropTypes.object.isRequired,
+        username: PropTypes.string,
+        password: PropTypes.string,
+        remember: PropTypes.bool,
+        onUsernameChange: PropTypes.func,
+        onPasswordChange: PropTypes.func,
+        onRememberToggle: PropTypes.func,
+        onLogin: PropTypes.func,
+        locale: PropTypes.string
+    }
+
+    static defaultProps = {
+        locale: 'en'
+    }
+
+    render() {
+        const {theme, username, password, remember, onUsernameChange, onPasswordChange, onRememberToggle, onLogin, locale} = this.props;
+        return (
+            <IntlProvider locale={locale} messages={getLocaleMessage(locale)}>
```

Comment:
Agree, putting this in Redux is a good solution because we can dynamically load it from the server.